api/src/backend/api/v1/views.py

```diff
@@ -1127,6 +1138,101 @@ def create(self, request, *args, **kwargs):
             },
         )
 
+    @extend_schema(
+        tags=["Scan"],
+        summary="Download ZIP report",
+        description="Returns a ZIP file containing the requested report",
+        request=ScanReportSerializer,
+        responses={
+            200: OpenApiResponse(description="Report obtanined successfully"),
+            404: OpenApiResponse(description="Report not found"),
+        },
```

Comment:
This should be autogenerated since this is detailed action view. Please avoid this kind of documentation details unless you add them for every viewset in the module.